### ISSUE TITLE
No access to `proxy_user` once connection is made

### ISSUE THREAD

**Issue by thrandale** [oracle/python-oracledb#250]
In cx_Oracle, when you connected with a proxy user, you were able to see that user in `connection.username`
```python
>>> import cx_Oracle
>>> connection = cx_Oracle.connect("sessionuser[proxyuser]/pw@db")
>>> print(connection.username)
sessionuser[proxyuser]
```
Though "hacky", you could parse this output to get the proxy user.

In oracledb, when you connect with a proxy user, there is no way to access the proxy user, as it is not included in `connection.username`
```python
>>> import oracledb
>>> oracledb.init_oracle_client()
>>> connection = oracledb.connect(
...     user="sessionuser",
...     proxy_user="proxyuser",
...     password="pw",
...     dsn="db",
... )
>>> print(connection.username)
sessionuser
```

It would be great to be able to access the connected proxy user trough either the `connection.username` or even better a `connection.proxy_user`

### Info
-  python -> 3.11.6
-  oracledb -> 1.4.2

**Comment by anthony-tuininga** [comment]
Our main focus at this point is on the support for asyncio but I will consider this small enhancement as well if I have an opportunity!

**Comment by anthony-tuininga** [comment]
I have pushed a patch that should implement this enhancement. If you are able to [build from source](https://python-oracledb.readthedocs.io/en/latest/user_guide/installation.html#installing-from-source-code) you can verify that it works for you.

**Comment by anthony-tuininga** [comment]
The patch has been included in version 2.0.0 which was just released.
